ext/curl/curl.stub.php

```diff
@@ -35,6 +35,8 @@ function curl_exec(CurlHandle $handle): string|bool {}
 
 function curl_file_create(string $filename, ?string $mime_type = null, ?string $posted_filename = null): CURLFile {}
 
+function curl_file_create_from_string(string $data, string $posted_filename, ?string $mime_type = null): CURLStringFile {}
```

Comment:
It might make sense to not add this function. It's 2021, people should know how to write `new CURLStringFile("xyz")` by now.